paddle/optimizer/Tensor.h

```diff
@@ -0,0 +1,49 @@
+#ifndef PADDLE_OPTIMIZER_TENSOR_H_
```

Comment:
we will change that and follow the google c++ style. 
https://google.github.io/styleguide/cppguide.html#The__define_Guard